data/explorations/android_interactions/Test all android interactions.yaml

```diff
@@ -0,0 +1,801 @@
+author_notes: ''
```

Comment:
Follow the filename pattern of other exploration yml file.

Reply:
Explorations in directories do have spaces.
- [Graph Modeling.yaml](https://github.com/oppia/oppia/blob/develop/data/explorations/modeling_graphs/Graph%20Modeling.yaml)

- [The Lazy Magician.yaml](https://github.com/oppia/oppia/blob/develop/data/explorations/binary_search/The%20Lazy%20Magician.yaml)

- [The Solar System.yaml](https://github.com/oppia/oppia/blob/develop/data/explorations/solar_system/The%20Solar%20System.yaml)

Reply:
@DubeySandeep PTAL

Reply:
It's weird that we have conflicts with the filename pattern in the dir. Anyway, thanks for checking! :)